runtime/src/main/java/org/corfudb/protocols/wireprotocol/ClientHandshakeHandler.java

```diff
@@ -1,5 +1,6 @@
 package org.corfudb.protocols.wireprotocol;
 
+import com.google.protobuf.TextFormat;
```

Comment:
There's a couple of instances in `NettyCorfuMsgEncoder` as well.

Reply:
Added